bindata/manifests/machine-config/files/configure-switchdev.sh.yaml

```diff
@@ -39,10 +63,6 @@ contents:
       # set PF to switchdev mode
       devlink dev eswitch set pci/${pci_addr} mode switchdev
 
-      # reset the pf name
-      new_name=$(ls /sys/bus/pci/devices/${pci_addr}/net/)
-      ip link set ${new_name} down
-      ip link set ${new_name} name ${names[i]}
       ip link set ${names[i]} up
```

Comment:
`${names[i]}`  could be a list of `pfName + vfRepNames`, is it expected?

Reply:
no, the names where collected in the first loop before the representors created